plugin/discovery/meta.go

```diff
@@ -0,0 +1,28 @@
+package discovery
+
+import (
+	"github.com/blang/semver"
```

Comment:
I see we have this in vendor already. Does it provide a significant benefit over our own `github.com/hashicorp/go-version`?